lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -121,13 +123,30 @@ def #{configuration[:column]}=(position)
 
             after_commit 'remove_instance_variable(:@scope_changed) if defined?(@scope_changed)'
 
-            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{connection.quote_column_name(configuration[:column])} IS NOT NULL}) }
+            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{quoted_position_column} IS NOT NULL}) }
+
+            def self.decrement_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} - 1))
+            end
+
+            def self.increment_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} + 1))
+            end
           EOV
 
           if configuration[:add_new_at].present?
             self.send :before_create, :"add_to_list_#{configuration[:add_new_at]}"
           end
+        end
 
+        private
+
+        def update_all_with_touch(updates)
+          attrs = new.send(:timestamp_attributes_for_update_in_model)
```

Comment:
Since this is very unlikey to change (could it ever?), perhaps we should be caching the timestamp attributes that we're going to update when `acts_as_list` is invoked?